django_evaluation/settings/local.py

```diff
@@ -273,20 +334,24 @@ def _set_favicon(html_color: str, project_root: Path) -> None:
     },
 }
 
-HOME_DIRS_AVAILABLE = web_config.get("HOME_DIRS_AVAILABLE", False)
+HOME_DIRS_AVAILABLE = False
```

Comment:
So you want to get rid of them eventually?

Reply:
It's not evaluated in the deployment. I don't even know if this would work if we set that to true. We are not allowed to mount the home dirs on the VM's - at least that used to be the case with xrootfs. Might be that with direct lustre mounts that's a different story now. But I think we should not reintroduce complexity.